metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
```

Comment:
@clouserw Is the official Mozilla Engagement Ratio formula finished? Last I saw it was a draft in a Google Doc somewhere.